docs: update canon-2 instructions (#1626)
umee-network/umee: networks/canon-2/README.md

# Canon-2

> Canon-2 is a long-running testnet mirroring mainnet functionality. We use it also as the last test before the mainnet upgrade for the next major version.

Reviewer: ```suggestion
Canon-2 is a long-running testnet mirroring mainnet functionality. We use it also as the last test before mainnet when doing chain upgrades.
```

Note: Additional steps required to become a validator

> ### config.toml changes

Reviewer: ```suggestion
### config.toml changes

(Trusted height and had should be replaced with a more recent block)
```
